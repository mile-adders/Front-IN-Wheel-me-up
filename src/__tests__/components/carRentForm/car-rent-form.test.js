  
/* eslint-disable no-undefined */
/* eslint-disable no-unused-vars */
import React from 'react';
// import {shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';
import CarRentForm from '../../../components/carRentForm/car-rent-form.js';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('<Form />', ()=>{
//   it('Form exists at the home page of our application',()=>{
//     let app = shallow(<CarRentForm />);
//     expect(app.find('button').exists()).toBeTruthy();
//     expect(app.find('input').exists()).toBeTruthy();
//     expect(app.find('form').exists()).toBeTruthy();

  //   });
  //   it('GET Method does a GET request from any API',()=>{
  //     let app = mount(<CarRentForm />);
  //     let button = app.find('button');
  //     button.simulate('click');
  //     expect(app.state('Headers')).toBe(undefined);
  //     expect(app.state('Response')).toBe(undefined);
  //   });
  it('renders correctly', () => {
    const tree = renderer.create(<CarRentForm />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});