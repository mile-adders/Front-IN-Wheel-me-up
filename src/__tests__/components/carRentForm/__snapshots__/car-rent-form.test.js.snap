// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input
      className="useInput"
      name="user"
      onChange={[Function]}
      placeholder="please enter your name"
      type="text"
      value=""
    />
    <input
      className="useInput"
      name="email"
      onChange={[Function]}
      placeholder="please enter your email"
      type="text"
      value=""
    />
    <input
      className="useInput"
      name="car"
      onChange={[Function]}
      placeholder="please choose your car"
      type="text"
      value=""
    />
    <input
      className="useInput"
      name="dailyRentTime"
      onChange={[Function]}
      placeholder="please enter when you want to rent"
      type="text"
      value=""
    />
    <input
      className="useInput"
      name="dateOut"
      onChange={[Function]}
      placeholder="please enter when you want to return the car"
      type="text"
      value=""
    />
    <input
      className="useInput"
      name="payment"
      onChange={[Function]}
      placeholder="please enter your payment method"
      type="text"
      value=""
    />
    <button
      className="submit"
      type="submit"
    >
      Go Rent!!
    </button>
  </form>
</div>
`;
